stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -132,7 +132,7 @@ end
 
         #pivoted upper Cholesky
         if eltya != BigFloat
-            cpapd = cholesky(apdh, Val(true))
+            cpapd = cholesky(apdh, RowMaximum())
```

Comment:
I'm wondering if we should maintain some tests that use `cholesky(apdh, Val(true))` to ensure that the deprecations work and to avoid accidental breakage of later?
```suggestion
            cpapd = cholesky(apdh, RowMaximum())
```

Reply:
That makes the tests fail, see my first commit and its fix in the second.

Reply:
Also, we don't have that for `lu` and `qr`. Seeing the few "fix breakage" PRs in downstream packages made me feel bad about deprecating these functions, though there were no objections against that. Maybe we shouldn't officially deprecate them but only redirect them and list those functions in the `deprecated.jl` list?

Reply:
Surprisingly, in my local build I don't even get a deprecation warning.

Reply:
> That makes the tests fail

I see. So I guess CI runs with `--depwarn=error`?

> Surprisingly, in my local build I don't even get a deprecation warning.

Do you run with `--depwarn=yes`?

Reply:
I just start the locally built Julia and then `using LinearAlgebra; A = rand(3,3); A = A'A; cholesky(A, Val(true))`. So, I wasn't referring to the tests, just to "normal usage".

Reply:
Yeah but the default now is `--depwarn=no` so users won't see these unless you set `--depwarn=yes` on startup.